### PR TITLE
Use cannonical ordering of PHPDoc

### PR DESCRIPTION
**Pull request by AJenbo**
This rule makes sure that PHPDoc is always written in the same order.

The only case where the project does not already follow this is a few cases of `@throws`.

### CODE REVIEW

**Comment by TheCartpenter** [comment]
Also the Index methods in PHPDoc.

**Comment by AJenbo** [comment]
> Also the Index methods in PHPDoc.

You appear to be talking about something unrelated to this PR.

**Comment by TheCartpenter** [comment]
> > Also the Index methods in PHPDoc.
> 
> You appear to be talking about something unrelated to this PR.

It's related. Each ```index()``` method have been modified as well on your behalf.

**Comment by AJenbo** [comment]
You are not making your self clear. None of the changes here touch any code and are purly in the PHPDoc so no methods where changed, let alone all `index()` methods.

**Comment by TheCartpenter** [comment]
> You are not making your self clear. None of the changes here touch any code and are purly in the PHPDoc so no methods where changed, let alone all `index()` methods.

Of course not. That's why lines line these have been added, since: https://github.com/opencart/opencart/commit/a84375a59f5c8d21f2932b25a9a4ab7bec4531e2#diff-d008a8c676ee59010961cb4c5e9b44e50b65a92f09ef5ee62472b3797b09395eR11 .

**Comment by TheCartpenter** [comment]
@danielkerr made it clear that we don't need the index put with PHPDoc.

**Comment by AJenbo** [comment]
> > You are not making your self clear. None of the changes here touch any code and are purly in the PHPDoc so no methods where changed, let alone all `index()` methods.
> 
> Of course not. That's why lines like these have been added, since: [a84375a#diff-d008a8c676ee59010961cb4c5e9b44e50b65a92f09ef5ee62472b3797b09395eR11](https://github.com/opencart/opencart/commit/a84375a59f5c8d21f2932b25a9a4ab7bec4531e2#diff-d008a8c676ee59010961cb4c5e9b44e50b65a92f09ef5ee62472b3797b09395eR11) .

That's not part of this PR, you are in fact commenting in the wrong place.

> @danielkerr made it clear that we don't need the index put with PHPDoc.

I simply followed what Daniel did here https://github.com/opencart/opencart/commit/9d9f4fa97ec916c608fb58690748602cb428dbfa based on his comment here https://github.com/opencart/opencart/pull/6196#issuecomment-347104018

**Comment by TheCartpenter** [comment]
> > > You are not making your self clear. None of the changes here touch any code and are purly in the PHPDoc so no methods where changed, let alone all `index()` methods.
> > 
> > 
> > Of course not. That's why lines like these have been added, since: [a84375a#diff-d008a8c676ee59010961cb4c5e9b44e50b65a92f09ef5ee62472b3797b09395eR11](https://github.com/opencart/opencart/commit/a84375a59f5c8d21f2932b25a9a4ab7bec4531e2#diff-d008a8c676ee59010961cb4c5e9b44e50b65a92f09ef5ee62472b3797b09395eR11) .
> 
> That's not part of this PR, you are in fact commenting in the wrong place.
> 
> > @danielkerr made it clear that we don't need the index put with PHPDoc.
> 
> I simply followed what Daniel did here [9d9f4fa](https://github.com/opencart/opencart/commit/9d9f4fa97ec916c608fb58690748602cb428dbfa) based on his comment here [#6196 (comment)](https://github.com/opencart/opencart/pull/6196#issuecomment-347104018)

Ok, I see he added it later on anyways after it was mentioned. We can disregard this, then.

**Comment by danielkerr** [comment]
I prefer the phpdoc

**Comment by AJenbo** [comment]
> I prefer the phpdoc

Can you be a bit more specific? It's hard to follow without a bit more context. I don't even know if you are replying to @TheCartpenter or something about this PR. My best guess is that you prefer a summery in the PHPDoc for `index()` methods on controllers.

**Comment by TheCartpenter** [comment]
> I prefer the phpdoc

It has now been completed: https://github.com/opencart/opencart/pull/13301 .

**Comment by AJenbo** [comment]
Deleted comments that where for a different PR
